data/sql/updates/pending_db_world/rev_1684589406305000945.sql

```diff
@@ -0,0 +1,10 @@
+--
+
+UPDATE `creature_addon` SET `auras` = '18950' WHERE `guid` IN (135170, 135171, 135172, 135173, 135174, 135175, 135345, 135346, 135347, 135348, 135349, 135350, 135351, 135352, 135353, 135354, 135355, 135356, 135357, 135358, 135359, 135360, 135361, 135362);
```

Comment:
It looks like there are some mistakes here.
135173 is an OLDWorld Trigger
135172 is a Shade of Aran Teleport Center

Reply:
```
SELECT * FROM `creature` WHERE `guid` IN (135170, 135171, 135172, 135173, 135174, 135175, 135345, 135346, 135347, 135348, 135349, 135350, 135351, 135352, 135353, 135354, 135355, 135356, 135357, 135358, 135359, 135360, 135361, 135362);
```

Reply:
They show up in Keira spawn addon of the selected creature

Reply:
which creature?

Reply:
oops no they dont